openapi/v2/openapi.json

```diff
@@ -124,19 +124,19 @@
           },
           {
             "name": "order.state",
-            "description": "State indicates the current state of the processing of the order.\n\n - UNSPECIFIED: Unspecified indicates that the value isn't set.\n - ACCEPTED: Accepted indicates that the order has been accepted by the system.\n - REJECTED: Rejected indicates that the order has been rejected by the system.\n - CANCELED: Rejected indicates that the order has been canceled by the user.\n - FULFILLED: Fulfilled indicates that the order has been successfully fulfilled.\n - FAILED: Failed indicates that fulfillment of the order failed.",
+            "description": "State indicates the current state of the processing of the order.\n\n - CLUSTER_ORDER_STATE_UNSPECIFIED: Unspecified indicates that the value isn't set.\n - CLUSTER_ORDER_STATE_ACCEPTED: Accepted indicates that the order has been accepted by the system.\n - CLUSTER_ORDER_STATE_REJECTED: Rejected indicates that the order has been rejected by the system.\n - CLUSTER_ORDER_STATE_CANCELED: Rejected indicates that the order has been canceled by the user.\n - CLUSTER_ORDER_STATE_FULFILLED: Fulfilled indicates that the order has been successfully fulfilled.\n - CLUSTER_ORDER_STATE_FAILED: Failed indicates that fulfillment of the order failed.",
```

Comment:
The description for `CLUSTER_ORDER_STATE_CANCELED` mentions "Rejected" instead of "Canceled"; this occurs throughout the document.

Reply:
Thanks @tzumainn, I am fixing it.

Reply:
That should be fixed now. Please take into account that the OpenAPI specifications are automatically generated from the gRPC specification. In addition there is a GitHub action that checks that the commit contains the re-generated code. So in general you don't need to review the OpenAPI files, only the gRPC specification.

Reply:
Makes sense - thanks!